underscore.js

```diff
@@ -1143,13 +1143,23 @@
     return eq(a, b, [], []);
   };
 
+  /**
```

Comment:
We don't use docblocks anywhere else in the code. Please use inline comments here.